hammerdb/send_pubkey.sh

```diff
@@ -6,7 +6,14 @@ set -u
 set -e
 # echo commands
 set -x
+# exit if left size of pipe failed
+set -o pipefail
 
 coordinator_private_ip=$1
-export pub_key=$(cat ~/.ssh/id_rsa.pub)
+pub_key=$(ssh-add -L | head -n 1)
```

Comment:
This script is run on the driver node. Here we are adding the public ip of driver node to the coordinator. If we use ssh agent here, it will be our local key since we will be connecting with ssh forwarding to the coordinator. So it seems correct to use the previous version here, what do you think?